### DISCUSSION TITLE
Global spacing presets

### DISCUSSION THREAD

**Discussion by Pixlkraft** [elementor/elementor#21535] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

With global spacing presets as part of site settings in Elementor, it would be a lot easier to manage spacing on flexbox containers to get consistent layouts. 

### Describe the solution you'd like

I would like it to be possible to add global spacing presets (padding/margin) as part of site settings in Elementor. Just like you can define global colors and fonts, it should also be possible to define spacing to get consistent layouts. This would make it possible to define several spacing variables to be used on eg. hero elements, card elements, small, medium, large containers etc.

### Describe alternatives you've considered

Right now it is only possible to define one single default padding value to your flexbox containers which is far from enough. The alternative is to manually set spacing on each element or define custom spacings with css classes.

**Comment by otomasz** [comment]
a much needed feature.  it will allow you to speed up work and stay ahead of competing solutions

**Reply by Pixlkraft** [reply]
Agreed. Elementor is said to have a built in global design system feature. But this is only partly true as spacing is a major element in a design system and this is currently not available. I don’t understand why this feature has been left out from the beginning. Hope this will be implemented soon 🤞

**Comment by 79ho3ein** [comment]
Hi

If I understand correctly, this is the **preset style** feature that Elementor is working on. I just know Elementor is working on it, it's not clear when it will be released.

Please see [here](https://github.com/elementor/elementor/issues/965).
